mmv1/third_party/terraform/tests/resource_compute_security_policy_test.go.erb

```diff
@@ -425,4 +425,97 @@ resource "google_compute_security_policy" "policy" {
 }
 `, spName)
 }
+<% end -%>
+
+<% unless version == 'ga' -%>
+func testAccComputeSecurityPolicy_withRedirectOptionsRecaptcha(spName string) string {
```

Comment:
These are the configs that include the redirect options, but we need an actual test that references these configs. See the functions above that set up `[]resource.TestStep{`

Reply:
The tests are just below them (with a capital first letter), do you prefer a different order? For some reason I'm having trouble getting them to work (getting `"message": "The resource 'projects/<project-id>/global/securityPolicies/tf-test-ifkn66nqvz' was not found"`).

Reply:
It was my bug, was missing flattening out the redirect_options back in...

Reply:
Just a point on the removing `redirect_options`. I actually doesn't make sense to test for, redirect_options can be only set for `redirect` type of rules (in fact, they don't make sense without the options). So removing it from a rule wouldn't be sensible.

Reply:
Ok, can you add an update test then to at least update part of the redirect_options config?

Reply:
Yes, I can add an update test easily.

Reply:
The test added and tests are passing.